### ISSUE TITLE
Trying to bind the Macrokey command to a macro prints that the command does not exist

### ISSUE THREAD

**Issue by floral-qua-floral** [Matts/MacroKey#66]
**Issue description**
Binding "/macrokey layer toggle" to a macro does not work. Attempting to use such a macro will output to the chat that the command does not exist.

**To Reproduce**
1. Load the mod.
2. Create at least two macrokey layers
3. Run the command "/macrokey layer toggle" manually to make sure it is working
4. Copy the command and paste it into a Macrokey macro
5. Press the button and observe the output in chat.

**Expected behaviour**
The keybind should swap between Macrokey layers just the same as running the command does.

**Screenshots**
![2019-12-10_14 11 06](https://user-images.githubusercontent.com/49110090/70573923-c2a63a80-1b57-11ea-93d9-9cb91d80421d.png)
![2019-12-10_14 11 12](https://user-images.githubusercontent.com/49110090/70573934-c5089480-1b57-11ea-9b85-f650394a4da5.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Forge version: 1.12.2-v14.23.5.2847
 - MacroKey Version: 2.0.3.202
 - Release Type: Release

**Downloaded From**
CurseForge

**Additional context**
This prevents toggle keybinds from being used.

**Comment by mchorse** [comment]
As far as I know, /macrokey command is client side, and MacroKey doesn't support client side commands. You can check that by Tabbing in the chat and seeing whether the name of the command is gray as opposed to white.

**Comment by Matts** [comment]
@mchorse is correct here. Currently, we do not support client-side commands. I do have this on the backlog IIRC (if not I will add an item). Been very busy with work and school the past few months, hope to be able to have time to come back to modding during my Christmas vacation (starting to miss it, and still have an almost ready 2.1 version I promised to bring out almost a year ago). Will keep the issue open until I have addressed it.

Thanks for choosing MacroKey

~Matt
